website/app.js

```diff
@@ -144,7 +145,14 @@ export async function main() {
     axis: {
       x: {
         type: "category",
-        categories: sha1SuffixList
```

Comment:
A. This looks like a merge error.
B. Why are the sha1 suffixes used and not prefixes?

Reply:
fixed